test/ffmpeg-qsv/transcode/vc1.py

```diff
@@ -0,0 +1,73 @@
+##
+### Copyright (C) 2018-2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ..util import *
+from .transcoder import TranscoderTest
+
+spec = load_test_spec("vc1", "transcode")
+class to_avc(TranscoderTest):
+  @slash.requires(have_ffmpeg_h264_qsv_encode)
+  @slash.requires(have_ffmpeg_vc1_qsv_decode)
+  @slash.parametrize(*gen_transcode_1to1_parameters(spec, "avc", "hwhw"))
+  @platform_tags(set(VC1_DECODE_PLATFORMS) & set(AVC_ENCODE_PLATFORMS))
+  def test_hwhw_1to1(self, case):
+    vars(self).update(spec[case].copy())
+    vars(self).update(
+      dstextension = 'h264',
+      case         = case,
+      mode         = 'hwhw',
+      ffencoder    = 'h264_qsv',
+      ffdecoder    = 'vc1_qsv',
+    )
+    self.transcode_1to1()
+
+class to_hevc(TranscoderTest):
+  @slash.requires(have_ffmpeg_hevc_qsv_encode)
+  @slash.requires(have_ffmpeg_vc1_qsv_decode)
+  @slash.parametrize(*gen_transcode_1to1_parameters(spec, "hevc", "hwhw"))
+  @platform_tags(set(VC1_DECODE_PLATFORMS) & set(HEVC_ENCODE_8BIT_PLATFORMS))
+  def test_hwhw_1to1(self, case):
+    vars(self).update(spec[case].copy())
+    vars(self).update(
+      dstextension = 'h265',
+      case         = case,
+      mode         = 'hwhw',
+      ffencoder    = 'hevc_qsv',
+      ffdecoder    = 'vc1_qsv',
+    )
+    self.transcode_1to1()
+
+class to_mjpeg(TranscoderTest):
+  @slash.requires(have_ffmpeg_mjpeg_qsv_encode)
+  @slash.parametrize(*gen_transcode_1to1_parameters(spec, "mjpeg", "hwhw"))
```

Comment:
missing `@slash.requires(have_ffmpeg_vc1_qsv_decode)`

Reply:
added it